docs/manual/java/releases/Migration15.md

```diff
@@ -114,12 +114,20 @@ my-database {
 }
 ```
 
-
-
 ## Service Discovery
 
 When opting in to Akka Cluster Bootstrapping as a mechanism for Cluster formation you will have to setup a [[Service Discovery|Cluster#Akka-Discovery]]  method for nodes to locate each other.
 
+## Production Settings
+
+New defaults have been added to the Lagom clustering configuration. 
+
+The first deafult configuration concerns how long a node should try to join an cluster. This is configured by the setting `akka.cluster.shutdown-after-unsuccessful-join-seed-nodes`. This is an existing Akka Cluster setting that is by default set to `off`, but Lagom will redefine it to 60 seconds. After that period, the Actor System will shutdown if it fails to join a cluster.
```

Comment:
I find this sentence confusing: `This is an existing Akka Cluster setting that is by default set to `off`, but Lagom will redefine it to 60 seconds.``

Instead, I think we should only mention the default value in Lagom:

`Lagom will default this value to 60 seconds.`

I'm not entirely sure, but is this value too low in the case of big clusters? If so, I'd mention how this value and the cluster size are correlated.

Reply:
I don't think the value is low for big clusters. To my understanding, the timeout can only quick in when it starts the joining process. Before that phase, it is discovering the seed-nodes.  

The discovering phase can indeed take some time because it must respect the number of minimal seed-nodes and the size of the cluster will, of course, affect it.